perfkitbenchmarker/cloudstack/cloudstack_network.py

```diff
@@ -0,0 +1,142 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import network
+from perfkitbenchmarker import vm_util
+from perfkitbenchmarker.cloudstack import util
+
+flags.DEFINE_string('cs_network_offering',
+                    'DefaultIsolatedNetworkOfferingForVpcNetworksNoLB',
+                    'Name of the network offering')
+
+flags.DEFINE_string('cs_vpc_offering',
+                    'Default VPC offering',
+                    'Name of the VPC offering')
+
+flags.DEFINE_boolean('cs_use_vpc', True,
+                     'Use VPC to create networks')
+
+
+FLAGS = flags.FLAGS
+
+
+class CloudStackFirewall(network.BaseFirewall):
```

Comment:
Yes please rebase from master.  All development is now done directly off
master.  We are going to delete the dev branch here soon.

On Wed, Oct 14, 2015 at 7:27 AM, Syed Mushtaq Ahmed <
notifications@github.com> wrote:

> In perfkitbenchmarker/cloudstack/cloudstack_network.py
> <https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/pull/527#discussion_r42000047>
> :
>
> > +flags.DEFINE_string('cs_network_offering',
> > +                    'DefaultIsolatedNetworkOfferingForVpcNetworksNoLB',
> > +                    'Name of the network offering')
> > +
> > +flags.DEFINE_string('cs_vpc_offering',
> > +                    'Default VPC offering',
> > +                    'Name of the VPC offering')
> > +
> > +flags.DEFINE_boolean('cs_use_vpc', True,
> > +                     'Use VPC to create networks')
> > +
> > +
> > +FLAGS = flags.FLAGS
> > +
> > +
> > +class CloudStackFirewall(network.BaseFirewall):
>
> Do you want me to rebase from the master?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/pull/527/files#r42000047>
> .
>



-- 
Anthony F. Voellm (aka Tony)
Google Voice:  (650) 516-7382
<https://www.google.com/voice/b/0?pli=1#phones>
Blog: http://perfguy.blogspot.com/
Benchmark like a pro... PerfKitBenchmarker
<https://github.com/GoogleCloudPlatform/PerfKitBenchmarker>